tests/python_keep_opening_on_previous_line/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,243 @@
+my_long_variable_aaaaaaaaaaaaaaaaaaaaaa = my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_aaaaaaaaaaaaaaaaaaaaaa = 1 / my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_1_aaaaaaaaaaaaaaaaaaaaaa = my_long_variable_2_aaaaaaaaaaaaaaaaaaaaaa = my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_1_aaaaaaaaaaaaaaaaaaaaaa = my_long_variable_2_aaaaaaaaaaaaaaaaaaaaaa = my_long_variable_3_aaaaaaaaaaaaaaaaaaaaaa = my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+a = [long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]
+
+a = b = [long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]
+
+my_long_variable_1_aaaaaaaaaaaaaaaaaaaaaa = my_long_variable_2_aaaaaaaaaaaaaaaaaaaaaa = [long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]
+
+foo(bar(baz([long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa])))
+
+foo(bar(baz([long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa], 1)))
+
+foo(bar(baz([long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]), 1))
+
+foo(bar(baz([long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa])), 1)
+
+long_func_1_aaaaaaaaaaaaa(long_func_2_aaaaaaaaaaaaaaaaa(long_func_3_aaaaaaaaaaaaaa(1)))
+
+def foo():
+    return [long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]
+
+
+def foo():
+    yield [long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+my_long_variable_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_aaaaaaaaaaaaaaaaaaaaaa = \\
+    1 / my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_1_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_variable_2_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_1_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_variable_2_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_variable_3_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+a = [
+    long_item_1_aaaaaaaaaaaa,
+    long_item_1_aaaaaaaaaaaa,
+    long_item_1_aaaaaaaaaaaa
+]
+
+a = b = [
+    long_item_1_aaaaaaaaaaaa,
+    long_item_1_aaaaaaaaaaaa,
+    long_item_1_aaaaaaaaaaaa
+]
+
+my_long_variable_1_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_variable_2_aaaaaaaaaaaaaaaaaaaaaa = [
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa
+    ]
+
+foo(bar(baz([
+    long_item_1_aaaaaaaaaaaa,
+    long_item_1_aaaaaaaaaaaa,
+    long_item_1_aaaaaaaaaaaa
+])))
+
+foo(bar(baz(
+    [
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa
+    ],
+    1
+)))
+
+foo(bar(
+    baz([
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa
+    ]),
```

Comment:
It seems like your goal is to not have `[]` be on its own line. You don't need conditionalGroup for this. What you can do when you are printing a function call is to look at the type of the arguments and if there's a single argument which is an array, then output something different. The whole game of prettier is from the ast to figure out what IR to output such that you get the right thing at the end.